examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -119,10 +137,11 @@ typedef bmpl::vector<
     particlePusher<UsedParticlePusher>,
     shape<UsedParticleShape>,
     interpolation<UsedField2Particle>,
-    current<UsedParticleCurrentSolver>
-    #if(PARAM_IONIZATION==1)
```

Comment:
nana that's fine, just keep `PARAM_IONIZATION` here.
the example I gave you was for an other case. sorry for the confusion.